Resources/Textures/DeltaV/Structures/Power/Generation/solar_panel.rsi/meta.json

```diff
@@ -0,0 +1,12 @@
+{
+	"version":1,
+	"license":"CC-BY-SA-3.0",
+	"copyright":"Taken from https://github.com/discordia-space/CEV-Eris/blob/d1e0161af146835f4fb79d21a6200caa9cc842d0/icons/obj/power.dmi and modified. Modified by Radezolid for random solar spawner",
+	"size":{"x":32,"y":32},
+	"states":
+	[
+		{
+			"name": "random_solar"
+		}
+	]
+}
```

Comment:
untroll all the json formatting so its like other files

Reply:
I guess, need DN to sign off on this